package.json

```
@@ -48,7 +48,7 @@
 		"webpack": "^4.40.0 || ^5.0.0"
 	},
 	"dependencies": {
-		"esbuild": "^0.19.0",
+		"esbuild": "^0.19.11",
```

```suggestion
		"esbuild": "^0.19.0",
```

Revert this back.
The lock file can use the latest esbuild, but this specifies the supported range of esbuild that users can install.